pom.xml

```diff
@@ -79,6 +79,14 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-config</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-bootstrap</artifactId>
+		</dependency>
```

Comment:
I am currently trying to wrap my head around this dependency. I think it refers to https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#config-first-bootstrap.

If I get things right, then with this dependency, we would automatically being connecting to a localhost:8888 trying to fetch there. This is a somewhat incompatible behavior, because we don't know in which environment Promregator is running: It may be that other spring-boot applications are using a local config server, but because Promregator did not in the past, we would be suddenly taking configuration values from there, potentially causing confusion. If with bad luck, this could even end up in a security flaw being introduced (hopefully, this wouldn't happen, though).

Essentially, what is currently on my mind is to facilitate this feature, but to not have it enabled by default. I am currently asking myself, if the option of using [Spring Boot Config Data Import](https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#config-data-import) isn't the better way.